modules/serverless-tech-preview-features.adoc

```diff
@@ -110,5 +110,15 @@ The following table provides information about which {ServerlessProductName} fea
 |GA
 |GA
 
+|Pipelines-as-code
+|-
+|-
+|TP
+
+|`new-trigger-filters` feature
```

Comment:
Either remove the word "feature", or substitute is with something that tells the reader what `new-trigger-filters` is.

Reply:
removed